shipyard-controller/handler/sequencedispatcher.go

```diff
@@ -119,7 +124,6 @@ func (sd *SequenceDispatcher) dispatchSequence(queuedSequence models.QueueItem)
 	// first, check if the sequence is currently paused
 	if sd.eventQueueRepo.IsSequenceOfEventPaused(queuedSequence.Scope) {
 		log.Infof("Sequence %s is currently paused. Will not start it yet.", queuedSequence.Scope.KeptnContext)
-		return ErrSequenceBlocked
```

Comment:
is this on purpose?

Reply:
Good catch - this was not supposed to be removed